cli/sawtooth_cli/block.py

```diff
@@ -46,6 +46,7 @@ def add_block_parser(subparsers, parent_parser):
         '--format',
         action='store',
         default='default',
+        choices=['csv', 'json', 'yaml'],
```

Comment:
It worked without it, because to trigger 'default', user simply left off the flag/option altogether. However, putting it in.